tests/unit/OptionsListUtilsTest.js

```diff
@@ -522,6 +522,13 @@ describe('OptionsListUtils', () => {
         expect(results.userToInvite).not.toBe(null);
         expect(results.userToInvite.login).toBe('+18003243233');
 
+        // When we use a search term for contact number, it shouldn't have any results if there are alphabets used
+        results = OptionsListUtils.getNewChatOptions(REPORTS, PERSONAL_DETAILS, [], '998243aaaa');
+
```

Comment:
Not a huge deal but let's stay consistent with the existing [When/Then](https://en.wikipedia.org/wiki/Given-When-Then) structure of the test
```suggestion
        // When we use a search term for contact number that contains alphabet characters
        results = OptionsListUtils.getNewChatOptions(REPORTS, PERSONAL_DETAILS, [], '998243aaaa');

        // Then we shouldn't have any results or user to invite
```

Reply:
@GItGudRatio 

`// Then we shouldn't have any results or user to invite` this is missed to add from the above suggestion.